Package.swift

```diff
@@ -87,7 +87,7 @@ let package = Package(
     .package(
       name: "GTMSessionFetcher",
       url: "https://github.com/google/gtm-session-fetcher.git",
-      "1.4.0" ..< "2.0.0"
+      "2.1.0" ..< "3.0.0"
```

Comment:
May want to be `"1.4.0" ..< "3.0.0"` to work with both Firebase 9 and 10.

Reply:
I could change it to that. I just matched how it currently is in the iOS SDK: https://github.com/firebase/firebase-ios-sdk/blob/master/Package.swift

Reply:
Please make it `"1.4.0" ..< "3.0.0"` so we can release it in a non-breaking change.